### ISSUE TITLE
Get access to the class without registering the custom element

### ISSUE THREAD

**Issue by daKmoR** [web-padawan/vanilla-colorful#1]
hey there 👋

that looks very nice 👍

any chance we can get an import that does not register the custom element? e.g. if you wanna use it with a scoped registry... or you need to allow 2 versions of the component to be loaded.

## strawman proposal

```js
import { ColorPickerHex } from 'vanilla-colorful/ColorPickerHex.js';
```

PS: could be archived by providing an export map 👍

**Comment by web-padawan** [comment]
Hi, thanks for the issue. Sounds like a good idea 👍 

I think the structure could look as follows:

- `lib/entrypoints/hex.js` - class that does not register the custom element
- `color-picker-hex.js` - custom element definition.

**Comment by web-padawan** [comment]
Released `0.1.1` version with this feature, see the [example](https://github.com/web-padawan/vanilla-colorful#base-classes).
